hr_recruitment_security/security/security.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <data>
+        <record id="group_hr_recruitment_officer_own_documents" model="res.groups">
+            <field name="name">Officer (own documents)</field>
+            <field name="category_id" ref="base.module_category_hr_recruitment"/>
+            <field name="implied_ids" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
```

Comment:
If you want to respect the dropdown, all groups must be stacked. I thought `Officer` was the highest permissions, but I see that is Manager. What are the differences?

I propose to stack groups that way, as if not, `Manager` will inherit `Office (own documents)` constraints and then you need to add extra record rules for giving again full access, but not sure if `Manager` implies too much permissions.

```
Officer
  |- Manager
     |- Manager (own documents)
```
```suggestion
            <field name="name">Manager (own documents)</field>
            <field name="category_id" ref="base.module_category_hr_recruitment"/>
            <field name="implied_ids" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]"/>
```
if we want to keep this structure:
```
Officer
  |- Officer (own documents)
     |- Manager
```
then we need to do this:
```suggestion
            <field name="name">Officer (own documents)</field>
            <field name="category_id" ref="base.module_category_hr_recruitment"/>
            <field name="implied_ids" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
        </record>
        <record id="hr_recruitment.group_hr_recruitment_manager" model="res.groups">
            <field name="implied_ids" eval="[(3, ref('hr_recruitment.group_hr_recruitment_user'), (4, ref('group_hr_recruitment_officer_own_documents'))]"/>
        </record>
        <record model="ir.rule" id="hr_job_group_hr_recruitment_manager_rule">
            <field name="name">View job positions (manager)</field>
            <field name="model_id" ref="hr_recruitment.model_hr_job" />
            <field name="domain_force">[(1, '=', 1)]</field>
            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]" />
        </record>
        <record model="ir.rule" id="hr_applicant_group_hr_recruitment_manager_rule">
            <field name="name">View applications (manager)</field>
            <field name="model_id" ref="hr_recruitment.model_hr_applicant" />
            <field name="domain_force">[(1, '=', 1)]</field>
            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]" />
```

Reply:
Thanks Pedro, it's a very good explanation about groups and inherit.